src/TestSuite/IntegrationTestCase.php

```diff
@@ -913,14 +913,15 @@ public function assertResponseEquals($content, $message = '')
      *
      * @param string $content The content to check for.
      * @param string $message The failure message that will be appended to the generated message.
+     * @param bool   $ignoreCase A flag to check whether we should ignore case or not.
      * @return void
      */
-    public function assertResponseContains($content, $message = '')
+    public function assertResponseContains($content, $ignoreCase = false, $message = '')
     {
```

Comment:
Let's keep it as the last parameter for now. Would I propose making another pull request with `ignoreCase` as the second parameter for the 4.0 branch?